payonidhi/if_else/leap_year.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+using namespace std;
+int main() {
+  int year;
+  cout<<"Enter any year"<<endl;
+  cin>>year;
+  if(year%4==0){
```

Comment:
Checkout this link to understand how to check leap year..
https://www.programiz.com/cpp-programming/examples/leap-year